src/DynamoCore/Scheduler/UpdateGraphAsyncTask.cs

```diff
@@ -148,14 +148,14 @@ protected override void HandleTaskCompletionCore()
                     // Clear node warning or info messages because if node is involved in new graph execution, message should be refreshed
                     if (node.State == ElementState.Warning)
                     {
-                        using (node.PropertyChangeManager.SetPropsToSuppress(nameof(NodeModel.ToolTipText), nameof(NodeModel.Infos), nameof(NodeModel.State)))
+                        using (node.PropertyChangeManager.SetPropsToSuppress(nameof(NodeModel.Infos), nameof(NodeModel.State)))
```

Comment:
Why are we removing tooltiptext? Is it replaced with an equivalent property on `nodeviewmodel`?

Reply:
ToolTipText is obsolete. Has been replaced by Infos.